www/securestorage.js

```diff
@@ -7,14 +7,7 @@ var _AES_PARAM = {
     cipher: 'aes'
 };
 
-var _checkCallbacks = function (success, error) {
```

Comment:
This has been moved to browser implementation as other platforms don't use it anymore (moved to executeNativeMethod)